static/src/stylesheets/inline/story-package-garnett.scss

```diff
@@ -77,17 +94,17 @@
     .fc-item.fc-item--pillar-arts.fc-item--type-comment,
     .fc-item.fc-item--type-feature
     {
-        background-color: $special-report-dark;
+        background-color: #383a3c; // This colour does not exist (in the palette)
```

Comment:
Is it worth extracting `#383a3c` out to a variable? (Partly just to give it a name, also because it's duplicated a few times here)